www/doc/1.7/using-cli.html

```diff
@@ -69,21 +69,25 @@ <h2 id="batch">Batch processing FPCores</h2>
   </p>
 
   <pre class="shell">herbie improve bench/tutorial.fpcore out.fpcore
-Starting Herbie on 3 problems (seed: 1809676410)...
-  1/3   [   1.799s]   39→ 0     Expanding a square
-  2/3   [   3.256s]    0→ 0     Commute and associate
-  3/3   [   0.937s]   29→ 0     Cancel like terms</pre>
+Starting Herbie on 3 problems (seed: 1551571787)...
+  1/3	[   0.882s]   30→ 0	Cancel like terms
+Warning: 24.7% of points produce a very large (infinite) output. You may want to add a precondition.
+See <https://herbie.uwplse.org/doc/1.6/faq.html#inf-points> for more.
+  2/3	[   1.721s]   29→ 0	Expanding a square
+Warning: Unsound rule application detected in e-graph. Results may not be sound.
+See <https://herbie.uwplse.org/doc/1.6/faq.html#unsound-rules> for more.
```

Comment:
I think your comments have convinced me we should rephrase this warning—it is way too scary for something actually pretty minor. In fact, perhaps we should drop this warning. (Discuss in today's meeting.)